src/ch07-01-creating-a-new-project.md

```diff
@@ -78,12 +78,15 @@ The files found in the `settings` directory are used for deployment:
 
 - `Devnet.toml` contains configuration that is used when running tests and in
   console mode.
-- `Mocknet.toml` is a template for deploying on a mocknet chain.
+- `Testnet.toml` is a template for deploying on a tesnet chain.
+- `Mainnet.toml` is a template for deploying on a mainnet chain.
 
 We will take a closer look at them later. Their contents are not important for
 our project but it is good to know what they are used for.
 
 Here is what the end result looks like. You can see your project files on the
 left, the terminal window on the bottom right, and your editor on the top right.
+*Please note that the directories shown in the image may differ with the current
+version.*
```

Comment:
Maybe you can replace the screenshot? Just be sure to hide anything unrelated.

Otherwise, maybe a note like this is alright.

Reply:
I don't think a screenshot here is valid. What I wanted to say was that this labels can change according to the version. Maybe it's not needed?

Reply:
Fair enough! Clarinet is rather in flux so I could see us having to replace the screenshot often. Then again the current one is _quite_ old.